jekyll/_ccie/backup.md

```diff
@@ -1,49 +1,51 @@
 ---
 layout: enterprise
 section: enterprise
-title: "Backing up CircleCI Enterprise"
+title: "Backing up CircleCI Data"
 category: [advanced-config]
 order: 8
 description: "How to regularly back up CircleCI Enterprise"
 ---
 
-This document will describe how to back up your CircleCI Enterprise application so that you can recover from accidental or unexpected loss of the VM containing all of your persistent CircleCI data (the services box) in the following sections:
+This document describes how to back up your CircleCI Enterprise application so that you can recover from accidental or unexpected loss of the Services machine VM that contains CircleCI data in the following sections:
 
 * TOC 
 {:toc}
 
-## Backing up DB data
+**Note:** If you are running CircleCI in an HA configuration, you must use standard backup mechanisms for the external datastores. See the [High Availability]({{site.baseurl}}/enterprise/high-availability/) document for more information.
 
-The primary, recommended mechanism for backing up your CircleCI data is to use VM snapshots of the virtual disk acting as the root volume for the services box (the machine that serves up the end-user-facing CircleCI UI).
+## Backing up the Database
 
-**Note:** If you are running CircleCI Enterprise in an HA configuration, you must use standard backup mechanisms for the external datastores. See the [High Availability]({{site.baseurl}}/enterprise/high-availability/) document for more information.
+The best practice for backing up your CircleCI data is to use VM snapshots of the virtual disk acting as the root volume for the Services machine. Backups may be performed without downtime as long the underlying virtual disk supports such an operation as is true with AWS EBS. There is a small risk, that varies by filesystem and distribution, that snapshots taken without a reboot may have some data corruption, but this is rare in practice. If zero downtime backups and robustness in the face of data corruption issues are required, then an [HA configuration]({{site.baseurl}}/enterprise/high-availability/) may be the best solution.
```

Comment:
Consider prefacing "The best practice" with something like "When running CircleCI with the databases on the Services machine, the best practice for..." -- this help set it apart from the "Best practice" of externalizing your databases.